static/js/components/PodcastCard.js

```diff
@@ -56,6 +56,9 @@ export default function PodcastCard(props: Props) {
           <div className="row podcast-author">
             <Dotdotdot clamp={2}>{podcast.offered_by}</Dotdotdot>
           </div>
+          <div className="row podcast-episode-count">
+            <Dotdotdot clamp={2}>{podcast.episode_count} Episodes</Dotdotdot>
```

Comment:
unless we have a podcast with a huge number of episodes I think we don't need the `Dotdotdot` here :smile: